university/src/main/java/nl/quintor/studybits/university/controllers/admin/SchemaController.java

```diff
@@ -17,22 +19,19 @@
 @RestController
 @RequestMapping("/{universityName}/admin/{userName}/schemas")
 @AllArgsConstructor(onConstructor = @__(@Autowired))
+@Slf4j
 public class SchemaController {
 
     private final UniversityService universityService;
     private final UserContext userContext;
     private final Mapper mapper;
 
-    private SchemaDefinitionModel toModel(SchemaDefinitionRecord record) {
-        return mapper.map(record, SchemaDefinitionModel.class);
-    }
-
     @GetMapping
     List<SchemaDefinitionModel> getSchemaDefinitions() {
         return universityService
                 .getSchemaDefinitions(userContext.currentUniversityName())
                 .stream()
-                .map(this::toModel)
+                .distinct()
```

Comment:
Is there no need for mapping the entity to a model anymore?

Reply:
Nope, as the getSchemaDefintions now returns models